draft-packages/modal/KaizenDraft/Modal/Presets/InputEditModal.tsx

```diff
@@ -64,24 +64,34 @@ const InputEditModal = ({
       automationId={automationId}
     >
       <div className={styles.modal} dir={localeDirection}>
-        <ModalHeader unpadded onDismiss={onDismiss}>
-          <div className={styles.header}>
+        <ModalHeader onDismiss={onDismiss}>
+          <div
+            className={classnames(styles.header, {
+              [styles.textAlignRTL]: localeDirection === "rtl",
+              [styles.padded]: !unpadded,
+            })}
+          >
             <ModalAccessibleLabel>
-              <Text tag="h1" style="zen-heading-3" inline>
+              <Heading tag="h2" variant="heading-2">
                 {title}
-              </Text>
+              </Heading>
             </ModalAccessibleLabel>
           </div>
         </ModalHeader>
-        <ModalBody unpadded>
-          <div className={styles.body} dir={localeDirection}>
+        <ModalBody>
+          <div
+            className={classnames(styles.body, { [styles.padded]: !unpadded })}
+            dir={localeDirection}
+          >
             {children}
           </div>
         </ModalBody>
         <ModalFooter
           actions={footerActions}
           appearance={type === "negative" ? "destructive" : "primary"}
           automationId={automationId}
+          variant={"input"}
```

Comment:
/nit You don't need the curly brackets here
```suggestion
          variant="input"
```